app/views/taxons/show_c.html.erb

```diff
@@ -1,5 +1,69 @@
-<%= render template: "taxons/show_a",
-  locals: {
-    presented_taxon: presented_taxon
-  }
+<% content_for :page_class, "taxon-page taxon-page--grid" %>
+<% content_for :is_taxon_with_subtopics, presented_taxon.show_subtopic_grid? %>
+<%=
+  render(
+      partial: 'common',
+      locals: {
+          presented_taxon: presented_taxon
+      }
+  )
 %>
+
+<div class="full-page-width-wrapper">
+  <div class="grid-row">
+    <div class="column-one-third">
+      <%= render "govuk_publishing_components/components/contents_list", {
+          hide_title: true,
+          contents: [
+              {
+                  text: "On this page:",
```

Comment:
We should probably add this to `en.yml` so it can be translated in future.

Reply:
👍